packages/message-relayer/src/test.ts

```diff
@@ -0,0 +1,17 @@
+import { ethers } from 'ethers'
```

Comment:
Is this a test or a `bin`?

Reply:
Just a temp testing script, lawl there goes my private key